packages/angular_devkit/build_angular/src/karma/code-coverage_spec.ts

```diff
@@ -27,12 +38,14 @@ describe('Karma Builder code coverage', () => {
 
     await run.stop();
 
+    await setTimeoutPromise(1000);
+
     const exists = host.scopedSync().exists(coverageFilePath);
-    expect(exists).toBe(true);
+    expect(exists).toBe(true, `${coverageFilePath} does not exist`);
 
     if (exists) {
       const content = virtualFs.fileBufferToString(host.scopedSync().read(coverageFilePath));
-      expect(content).toContain('polyfills.ts');
```

Comment:
karma-coverage by default does not check files that are not tested.
If users want coverage for all files, they could turn it on by specifying [`includeAllSources`](https://github.com/karma-runner/karma-coverage/blob/master/docs/configuration.md#includeallsources)